lib/interop.js

```diff
@@ -468,7 +468,6 @@ Interop.prototype.toUnifiedPlan = function(desc) {
                         // msid. Note that "recvonly" is a term only defined
                         // for m-lines.
                         uLine.msid = sources[ssrc].msid;
-                        uLine.direction = 'sendrecv';
```

Comment:
Yes, if I am not wrong, toUnifiedPlan and toPlanB funcions are just a translator, so they must keep the meaning between the input and the output SDPs, isn't it?
If we agree this point, the case that you say is working well. If you need sendrecv the "bLine prototype" should be sendrecv.